### DISCUSSION TITLE
imort-feeds how to add product attribute typeValues?

### DISCUSSION THREAD

**Discussion by zssvaidar** [atrocore/atrocore#520] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/30953478/202117366-482c4559-ad11-4f2f-80c7-1b7098dcc001.png)
![image](https://user-images.githubusercontent.com/30953478/202118290-3d53c182-6c09-4efc-9003-d3971b1314f7.png)

Our feed contains large number or products each with new AttributeType values, our target `attribute` has type List,
how can we add new values to attributeType values from import feed?

Ty for dealing with the issue

**Comment by rratsun** [comment, marked as answer]
Hello,

You should import an Attribute entity for this. As I see, now you import a Product entity. So firstly, try to import attributes and then import products. If you need automations here, please, buy Connector module.
